test/Faker/Provider/ProviderOverrideTest.php

```diff
@@ -6,13 +6,7 @@
 use Faker\Test\TestCase;
 
 /**
- * Class ProviderOverrideTest
- *
- * @package Faker\Test\Provider
- *
- * This class tests a large portion of all locale specific providers. It does not test the entire stack, because each
- * locale specific provider (can) has specific implementations. The goal of this test is to test the common denominator
- * and to try to catch possible invalid multi-byte sequences.
+ * Class ProviderOverrideTest.
  */
```

Comment:
Why remove the explanation?

Just repeating the class name doesn't add much value. This comment might as well be removed completely.